backend/src/services/slc-function/slc-function.entity.ts

```diff
@@ -87,6 +87,10 @@ class SLCFunction {
     });
   }
 
+  public updateUpdatedAt(): void {
+    this.updatedAt = new Date().toISOString();
+  }
+
   public setSourceCode(sourceCode: string): void {
     this.sourceCode = sourceCode;
   }
```

Comment:
```suggestion
   public setSourceCode(sourceCode: string): void {
    this.sourceCode = sourceCode;
    this.updatedAt = new Date().toISOString();
  }
```

Reply:
The more logic you put on your services, the more brittle your design will become